packages/web/src/assets/styles/type.css

```diff
@@ -160,5 +160,6 @@
   --font-medium: 500;
   --font-demi-bold: 600;
   --font-bold: 700;
+  --font-extra-bold: 800;
```

Comment:
figured we would need this at some point haha. did you have to add another font file for this or did it just work?